### PR TITLE
Get rid of Firebird

### PR DESCRIPTION
**Pull request by prepor**

### CODE REVIEW

File: xapix/README.md [hunk 1]
```diff
@@ -141,7 +140,7 @@ For the databases, make sure to adjust the following setting in "Settings > Conn
 
 Use `helm status xapix` to see the status of your install.
 
-1. Firebird, Phoenix and Shisa will run DB setup jobs which create the required DBs. If this fails, we might not have enough permissions to access the DB or the DB credentials might be wrong. Check the logs for more details.
+1. Clojud, Phoenix and Shisa will run DB setup jobs which create the required DBs. If this fails, we might not have enough permissions to access the DB or the DB credentials might be wrong. Check the logs for more details.
```
Comment (mreinsch) [inline xapix/README.md]: no clojud here

File: xapix/README.md [hunk 2]
```diff
@@ -28,26 +28,25 @@ Depending on the scale of your setup, Xapix can be set up with one shared Postgr
 
 The following services need a PostgreSQL database:
 
- 1. Firebird: to store data transformation configuration; storage: 10 GB; instance: depending on anticipated traffic; **critical for API access**
```
Comment (mreinsch) [inline xapix/README.md]: would be good to mention that clojud uses kafka and what storage requirements we have there. Also that there is an option to use existing kafka instance?

File: xapix/values.yaml [hunk 3]
```diff
@@ -122,11 +87,33 @@ shisa:
...
 clojud:
-  enabled: false
+  app:
+    # internal Kafka Streams id for Clojud
+    id: dev-clojud
+    # number of worker threads for a Clojud instance
+    threads: 5
+  kafka:
+    # clojud creates kafka topics with the following prefix
+    topicPrefix: dev-clojud
```
Comment (mreinsch) [inline xapix/values.yaml]: let's use `xapix-clojud` as default here

File: xapix/values.yaml [hunk 4]
```diff
@@ -122,11 +87,33 @@ shisa:
...
   api_secret:       # openssl rand -hex 10
 
-# for next generation firebird, still in testing
+
+#  Clojud is the API gateway which will be used by API users to access your APIs.
 clojud:
-  enabled: false
+  app:
+    # internal Kafka Streams id for Clojud
+    id: dev-clojud
```
Comment (mreinsch) [inline xapix/values.yaml]: let's use `xapix-clojud` or something as default here

File: xapix/values.yaml [hunk 5]
```diff
@@ -122,11 +87,33 @@ shisa:
...
+      -Xmx2048m
+
+  # Prometheus Service Monitor config; requires monitoring.coreos.com/v1 API
+  serviceMonitor:
+    enabled: false
+
+# if install == true it deploys basic kafka cluster
 kafka:
-  install: false
+  install: true
```
Comment (mreinsch) [inline xapix/values.yaml]: can we add the most common variables, such as replicaCount, persistence settings?

File: xapix/values.yaml [hunk 6]
```diff
@@ -122,11 +87,37 @@ shisa:
...
+  persistence:
```
Comment (mreinsch) [inline xapix/values.yaml]: can we also add `storageClass: default  # persistent volume class, needs to be created independently` ? It's cluster specific, so usually requires adjustment

And no replicaCount ?

Comment (prepor) [reply xapix/values.yaml]: I think it's a very rare case when someone could want something different than replicaCount = 3 in small / medium setups

**Pull request merged**